api/app/main.py

```
@@ -0,0 +1,137 @@
+        parse_start_time = time.time()
+        parsed_data = parse_resume_text(text, annotation_links)
+        parse_time = (time.time() - parse_start_time) * 1000
+        logger.info(f"Parsing complete for {file_key} in {parse_time:.2f}ms.")
+
+        return parsed_data
+
+    except HTTPException as http_exc:
+        logger.error(
+            f"HTTPException during processing {file_key}: Status={http_exc.status_code}, Detail={http_exc.detail}")
+        raise http_exc
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)